htdocs/bootstrap/css/custom-css.css

```diff
@@ -175,3 +175,10 @@ div.navbar div.container div.navbar-brand {
     to { transform: scale(1) rotate(360deg);}
 }
 
+/*
+ *      CSS for alert div
+ */
+.no-result-found-panel {
```

Comment:
In fact, this should be replace so it can be used by the breadcrumbs also. Both breadcrumbs and this panel inherit from the bootstrap alert class but feel the need to have their text white...